src/PostgREST/ApiRequest.hs

```diff
@@ -122,6 +122,10 @@ userApiRequest schema req reqBody =
               Nothing -> PayloadParseError "All lines must have same number of fields"
               Just json -> PayloadJSON json)
             (CSV.decodeByName reqBody)
+        Left "application/x-www-form-urlencoded" ->
```

Comment:
This seems like it should be handled in pickContentType and get a "Right" value from there